frontend/src/app/models/recipe.ts

```diff
@@ -0,0 +1,9 @@
+export default class Recipe {
```

Comment:
An interface or a type makes more sense for models

```suggestion
export default interface Recipe {
```